docs/onnx_model_zoo.md

```diff
@@ -0,0 +1,41 @@
+## KotlinDL ONNX Model Zoo
+
+| Model                     |     Android support      |
```

Comment:
Intellij suggests to reformat this table:
![image](https://user-images.githubusercontent.com/58706/196305300-1fd4a500-4eb1-41a7-aa2e-a2534a00590a.png)